### ISSUE TITLE
Indentation removed in bulleted list

### ISSUE THREAD

**Issue by algomaster99** [iterative/dvc.org#358]
Commit [a57596](https://github.com/iterative/dvc.org/commit/a5759608bc49c50fd8c304c2465becc31e7dd079) removes indentation from bulleted list.

### Issue

```
- Syntax highlighting in fenced code blocks should use the `usage` and `dvc`
custom languages:
```

### Fix

```
- Syntax highlighting in fenced code blocks should use the `usage` and `dvc`
  custom languages:
```

This can be fixed by running `npm run prettier-md`.

**Comment by ryokugyu** [comment]
Will you please screenshot those particular blocks and give me a brief review of it!

**Comment by algomaster99** [comment]
@ryokugyu 
### Issue
![Screenshot from 2019-05-20 19-55-14](https://user-images.githubusercontent.com/35191225/58029518-65568780-7b3a-11e9-8f73-472450395b51.png)

### After formatting
![Screenshot from 2019-05-20 20-04-19](https://user-images.githubusercontent.com/35191225/58029569-7dc6a200-7b3a-11e9-96c3-ef83e83981f4.png)

**Comment by algomaster99** [comment]
@ryokugyu 
### Issue
![Screenshot from 2019-05-20 19-55-14](https://user-images.githubusercontent.com/35191225/58029518-65568780-7b3a-11e9-8f73-472450395b51.png)

### After formatting
![Screenshot from 2019-05-20 20-04-19](https://user-images.githubusercontent.com/35191225/58029569-7dc6a200-7b3a-11e9-96c3-ef83e83981f4.png)

**Comment by shcheklein** [comment]
@ryokugyu @algomaster99 yep, that commit was too aggressive. Let's run prettier. We should check that it will indent code blocks under list items back (two spaces).

**Comment by algomaster99** [comment]
@shcheklein Yes, `npm run prettier-md` works. @ryokugyu Will you submit a PR? :smile:.

**Comment by ryokugyu** [comment]
I will submit pr for it. :D  @algomaster99

**Comment by jorgeorpinel** [comment]
Hello! You may want to take a look at #363 as you work on this one. Thanks

**Comment by shcheklein** [comment]
@ryokugyu hey, any updates on this?

**Comment by shcheklein** [comment]
@jorgeorpinel does your last PR fixes that? This one should be done for the most part. Except some code block under list items - not a big issue I think.

**Comment by jorgeorpinel** [comment]
Yes it was fixed in recent PRs including #378 e.g. https://github.com/iterative/dvc.org/pull/378/files#diff-ad6b7d6b56ae9694f56f7d353f1eb56aR47

Closing (:

**Comment by ryokugyu** [comment]
> @ryokugyu hey, any updates on this?

I didn't worked on it because of the issue of #363
